src/collections/events/addo/index.mdx

```diff
@@ -2,9 +2,10 @@
 title: "2020 All Day DevOps"
 date: 2020-11-12
 thumbnail: ./all-day-devops.png
-eurl: https://www.alldaydevops.com/addo-speakers/lee-calcote
+eurl: /community/events/2020-all-day-devops
 topic: event
 published: true
-speakers: [""]
 
 ---
+
+<h4 style="margin-bottom:1.25rem; font-weight: 700"><a href="https://www.alldaydevops.com/addo-speakers/lee-calcote">Event Link</a></h4>
```

Comment:
This can also be added to the `[Events-single](https://github.com/layer5io/layer5/blob/master/src/sections/Community/Event-single/index.js)` page. Instead of adding them to all the pages invidiually